extension/src/views/webView.ts

```diff
@@ -108,10 +108,16 @@ export abstract class WebView<T> implements Disposable {
     protected async getHead(nonce: string): Promise<string> {
         const cspSource = this.panel?.webview.cspSource ?? '';
 
+        const allowInsecure = vscode.workspace
+            .getConfiguration('privateExtensions')
+            .get<boolean>('allowInsecureContent');
+
+        const allowedProtocol = allowInsecure ? 'http' : 'https';
```

Comment:
If we only use `http:`, does that automatically include HTTPS, or would this then allow *only* insecure content? Reading https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/img-src didn't make that very clear to me, but it seems like you would need to use `img-src http: https:` to allow both secure and insecure content?

Reply:
Good point. I've just checked it on a real example and using `http:` we are getting both secure and insecure content loaded correctly.